pallets/automation-time/src/tests.rs

```diff
@@ -2494,192 +2606,403 @@ fn trigger_tasks_completes_some_xcmp_tasks() {
 #[test]
 fn trigger_tasks_completes_auto_compound_delegated_stake_task() {
 	new_test_ext(START_BLOCK_TIME).execute_with(|| {
-		get_funds(AccountId32::new(ALICE));
-		let before_balance = Balances::free_balance(AccountId32::new(ALICE));
+		let delegator = AccountId32::new(DELEGATOR_ACCOUNT);
+		get_funds(delegator.clone());
+		let before_balance = Balances::free_balance(delegator.clone());
 		let account_minimum = before_balance / 2;
 
-		let task_id = add_recurring_task_to_task_queue(
-			ALICE,
+		add_recurring_task_to_task_queue(
+			DELEGATOR_ACCOUNT,
 			vec![1],
 			SCHEDULED_TIME,
 			3600,
 			Action::AutoCompoundDelegatedStake {
-				delegator: AccountId32::new(ALICE),
-				collator: AccountId32::new(BOB),
+				delegator: delegator.clone(),
+				collator: AccountId32::new(COLLATOR_ACCOUNT),
 				account_minimum,
 			},
+			vec![],
 		);
 
 		LastTimeSlot::<Test>::put((LAST_BLOCK_TIME, LAST_BLOCK_TIME));
 		System::reset_events();
 
 		AutomationTime::trigger_tasks(Weight::from_ref_time(120_000));
+		let emitted_events = events();
 
-		let new_balance = Balances::free_balance(AccountId32::new(ALICE));
+		let new_balance = Balances::free_balance(delegator.clone());
 		assert!(new_balance < before_balance);
 		assert_eq!(new_balance, account_minimum);
-		let delegation_event = events()
+
+		emitted_events
+			.clone()
 			.into_iter()
-			.find(|e| match e {
-				RuntimeEvent::AutomationTime(
-					crate::Event::SuccesfullyAutoCompoundedDelegatorStake { .. },
-				) => true,
-				_ => false,
-			})
-			.expect("AutoCompound success event should have been emitted");
-		let execution_weight = MockWeight::<Test>::run_auto_compound_delegated_stake_task();
-		let fee = ExecutionWeightFee::get().saturating_mul(execution_weight.ref_time().into());
-		assert_eq!(
-			delegation_event,
-			RuntimeEvent::AutomationTime(crate::Event::SuccesfullyAutoCompoundedDelegatorStake {
-				task_id,
-				amount: before_balance
-					.checked_sub(account_minimum.saturating_add(fee))
-					.expect("Event should not exist if value is neg"),
-			})
-		);
+			.find(|e| matches!(e, RuntimeEvent::AutomationTime(crate::Event::TaskExecuted { .. })))
+			.expect("TaskExecuted event should have been emitted");
```

Comment:
The format of the `events` for verification is different from other tests. Please update the format to be consistent with others.

Reply:
Fixed